dealership/contracts.py

```diff
@@ -1,12 +1,50 @@
-class Contract(object):
-    pass
+from .customers import Customer, Employee
+from .vehicles import Car, Truck, Motorcycle
 
+class Contract(object):
+    CAR_INTEREST = 1.07
+    MOTORCYCLE_INTEREST = 1.03
+    TRUCK_INTEREST = 1.11
 
```

Comment:
Since `vehicle` and `customer` are found in both `BuyContract` and `LeaseContract`, you could move those to the Contract super class, so that you have them available in all sub-classes without repeating any code.

Reply:
u mean the initialization of Vehicle and Customer class  ?     
 self.vehicle = vehicle
 self.customer = customer

Reply:
Exactly, you got it!